docs/source/ser_deser.rst

```diff
@@ -0,0 +1,78 @@
+Serialization and Deserialization for torchao optimized models
```

Comment:
can we just call this file serialization?

Reply:
sure, also the title?

Reply:
yeah